PyInstaller/utils/hooks/qt/__init__.py

```diff
@@ -181,6 +160,30 @@ def __getattr__(self, name):
         # ... and return the requested attribute
         return getattr(self, name)
 
+    # Check whether we can use the new layout (e.g. PyQt5/Qt5, PyQt6/Qt6) instead of the old layout (PyQt5/Qt, PyQt6/Qt)
+    @staticmethod
+    def _can_use_new_layout(requirements: str, requirements_commercial: str, fallback: bool) -> bool:
```

Comment:
Two nitpicks here:

1. I would prefer if this was named just `_use_new_layout` (without can, because that implies an option, which is not really there). Same in the comment ("Check if we *can* use...") - we actually must use the new layout.

2. For the argument, I would prefer to see `package_basename`, `version`, and `fallback_value` (or `default_value`).

The helper would then be invoked like `self._use_new_layout("PyQt5", "5.15.4", False)`, and it would constructs both strings for `is_module_satisfies` on its own.

Regarding `fallback_value` vs just `fallback`, the latter implies to me that this is a switch that will enabled a fallback within the helper, as opposed to the fallback/default value returned by the helper.

Reply:
I should have addressed these points. Let me know if there's something else to do.